.circleci/config.yml

```diff
@@ -195,9 +195,14 @@ jobs:
       - image: cimg/openjdk:<< pipeline.parameters.java-tag >>
     steps: # a collection of executable commands
       - checkout # check out source code to working directory
-      - restore_cache: # restore the saved cache after the first run or if `pom.xml` has changed
+      - restore_cache: # restore the saved cache after the first run or if `THIRD-PARTY-LICENSES.txt` has changed
```

Comment:
It was like this, but shouldn't it be restore if the licenses file **hasn't** changed?

Okay, it's more like restore the license based cache if it hasn't changed, otherwise restore the most recent cache. The wording is a tad confusing, or I'm misunderstanding.

With your comments a few lines down, maybe you don't need this comment at all?

Reply:
> With your comments a few lines down, maybe you don't need this comment at all?

Probably this. 

The new way, we always restore some cache or other, it just might be less specific